models/user.py

```diff
@@ -56,7 +56,7 @@ class User(UserMixin, db.Model):
     role_id = db.Column(db.Integer, db.ForeignKey('roles.id'))
 
     def __init__(self, **kwargs):
-        super(User, self).__init__(**kwargs)
```

Comment:
**Problem:** you can use the super() function without any arguments to call the superclass's constructor without explicitly specifying the class name
**Suggestion:** `        return f'<Role \'{self.name}\'>'`